configs/s2ef/all-forceonly/forcenet/forcenet_basic.yml

```diff
@@ -0,0 +1,43 @@
+trainer: forces
+
+dataset:
+  - src: /checkpoint/electrocatalysis/relaxations/features/struct_to_energy_forces/train/200k/
```

Comment:
We should make these paths consistent with those in the other configs. Can we include the base config from one of the default ones in the repository?

Reply:
We should probably also change `all-forceonly/forcenet/forcenet_basic.yml` to `forcenet/forceonly.yml` (consistent with how we've done it for dimenet-plus-plus)

Reply:
Done